parsons/etl/etl.py

```diff
@@ -659,20 +658,12 @@ def unpack_nested_columns_as_rows(self, column, key="id", expand_original=False)
             orig.concat(melted_list)
             # Add unique id column by hashing all the other fields
             if "uid" not in self.columns:
-                if sys.version_info.minor >= 9:
-                    orig.add_column(
-                        "uid",
-                        lambda row: hashlib.md5(
-                            str.encode("".join([str(x) for x in row])), usedforsecurity=False
-                        ).hexdigest(),
-                    )
-                elif sys.version_info.minor < 9:
-                    orig.add_column(
-                        "uid",
-                        lambda row: hashlib.md5(  # nosec B324
-                            str.encode("".join([str(x) for x in row]))
-                        ).hexdigest(),
-                    )
+                orig.add_column(
+                    "uid",
+                    lambda row: hashlib.md5(
+                        str.encode("".join([str(x) for x in row])), usedforsecurity=False
+                    ).hexdigest(),
+                )
```

Comment:
IDK what this section of code is, but these changes seem out of scope of this PR?

Reply:
ohh I see, it's checking for the python version being 3.9 or higher